src/storybook/story-test-runner/extend-stories.ts

```diff
@@ -12,9 +12,30 @@ type StoryFile = { default: HermioneMetaConfig } & Record<string, HermioneStoryC
 
 export function extendStoriesFromStoryFile(stories: StorybookStory[]): StorybookStoryExtended[] {
     const storiesMap = getStoriesMap(stories);
-    const storyFile = getStoryFile(stories[0].absolutePath);
+    const storyPath = stories[0].absolutePath;
+    const storyFile = getStoryFile(storyPath);
     const withStoryFileExtendedStories = stories as StorybookStoryExtended[];
 
+    if (!storyFile) {
+        if (storyPath.endsWith(".tsx")) {
+            const msg = [
+                "[hermione-storybook]",
+                "reading .tsx story files is not supported.",
+                `"hermione" section is ignored in "${storyPath}"`,
+            ].join(" ");
+
+            console.warn(msg);
+        }
+
+        return withStoryFileExtendedStories.map(story => {
+            story.skip = false;
+            story.assertViewOpts = {};
+            story.browserIds = null;
+
+            return story;
+        });
```

Comment:
and normalize the stories here, so `extendStoriesFromStoryFile` always returns stories of the specified type